### PR TITLE
New commands

### PR DESCRIPTION
**Pull request by 0xPxt**
Implementation for sign_tx_hash and make_transfer_tx

### CODE REVIEW

**Comment by 0xPxt** [comment]
zemu tests failing on GH Actions because of the JS package, once [this PR](https://github.com/Zondax/ledger-kadena-js/pull/1) is merged everything should be ok.

File: tests_zemu/tests/testscases/transactions.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      creationTime: 1665722463,
+      ttl: '600',
+      nonce: '2022-10-14 04:41:03.193557 UTC',
+      recipient_chainId: 2,
+    },
+  },
+]
+
+export const TRANSACTIONS_LEGACY_TEST_CASES = [
+  // {
```
Comment (chcmedeiros) [inline tests_zemu/tests/testscases/transactions.ts]: Some commented testcases, not sure if it is supposed to

File: app/src/common/tx.c [hunk 2]
```diff
@@ -20,49 +20,72 @@
...
 
 uint32_t tx_get_buffer_length() { return buffering_get_buffer()->pos; }
 
 uint8_t *tx_get_buffer() { return buffering_get_buffer()->data; }
 
-const char *tx_parse() {
-    MEMZERO(&tx_obj, sizeof(tx_obj));
-
-    uint8_t err = parser_parse(&ctx_parsed_tx, tx_get_buffer(), tx_get_buffer_length(), &tx_obj);
+const char *tx_parse(tx_type_t tx_type) {
```
[the remark this answers is not in the document]
Comment (chcmedeiros) [reply app/src/common/tx.c:87]: You can change the name of the argument.

**Comment by chcmedeiros** [comment]
I think we should squash and merge into dev, and then point the PR with backward compatibility also to dev. 
Later we merge dev into main with cleaner history.

File: app/src/parser_impl.c [hunk 3]
```diff
@@ -180,12 +220,207 @@ parser_error_t parser_getValidClist(uint16_t *clist_token_index, uint16_t *num_a
...
+    char address[65] = {0};
+    uint16_t address_len = 0;
+    chunk_t chunks[MAX_FIELDS_IN_INPUT_DATA] = {0};
+
+    parser_readSingleByte(ctx, &tx_type);
+
+    for (int i = 0; i < MAX_FIELDS_IN_INPUT_DATA; i++) {
+        parser_readSingleByte(ctx, &chunks[i].len);
+        if (chunks[i].len > 0) {
+            parser_readBytes(ctx, (uint8_t **)&chunks[i].data, chunks[i].len);
```
Comment (chcmedeiros) [inline app/src/parser_impl.c]: Can we add checking for the return of these functions., so we can propagate the error.

**Review by chcmedeiros — APPROVED** [review]

**Comment by chcmedeiros** [comment]
LGTM! But let's have the JS one merged first so we can make sure Zemu is running smooth here :)

**Comment by emmanuelm41** [comment]
@0xPxt I have a question regarding the INS. Are those our INS, or the ones for retro compatibility? If those are our ones, I would name them without the KDA suffix. I would leave that KDA suffix for the backward compatibility INS.

**Pull request merged**